src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style.trim() !== '');
+
+  const stylesObject = styles.reduce((acc, style) => {
+    const [property, value] = style.split(':', 2)
+      .map((item) => item.trim());
+
+    if (property.length > 0) {
+      acc[property] = value;
+    }
+
+    return acc;
```

Comment:
The use of `reduce` here is appropriate for building up the styles object. However, you should check if the `value` is also non-empty before adding the property to the accumulator object to avoid adding invalid entries.